server/views/formPages/supervisor/review.html

```diff
@@ -19,7 +19,7 @@
 {% endblock %}
 
 {% block formItems %}
-
+{% set isFemale = user.activeCaseLoad.female %}
```

Comment:
Same question here

Reply:
This is used further down on line 148 but is in the wrong place.